IS7024FinalProjectBreweryArc/BreweryArc/Pages/Index.cshtml

```diff
@@ -78,51 +76,47 @@
-                <th>State</th>
-                <th>County Province</th>
-                <th>Postal Code</th>
-                <th>Country</th>
-                <th>Longitude</th>
-                <th>Latitude</th>
-                <th>Phone</th>
-                <th>Website Url</th>
-                <th>Update At</th>
-                <th>Created At</th>
-
-            </tr>
-            @{
-                
-                    <tr>
-                        <td>@brewDetail.Id</td>
-                        <td>@brewDetail.Name</td>
-                        <td>@brewDetail.BreweryType</td>
-                        <td>@brewDetail.Street</td>
-                        <td>@brewDetail.Address2</td>
-                        <td>@brewDetail.Address3</td>
-                        <td>@brewDetail.City</td>
-                        <td>@brewDetail.State</td>
-                        <td>@brewDetail.CountyProvince</td>
-                        <td>@brewDetail.PostalCode</td>
-                        <td>@brewDetail.Country</td>
-                        <td>@brewDetail.Longitude</td>
-                        <td>@brewDetail.Latitude</td>
-                        <td>@brewDetail.Phone</td>
-                        <td>@brewDetail.WebsiteUrl</td>
-                        <td>@brewDetail.UpdatedAt</td>
-                        <td>@brewDetail.CreatedAt</td>
-
-                    </tr>
-
-               
-            }
+            <thead>
+                <tr>
+                    <th>Name</th>
+                    <th>Brewery Type</th>
+                    <th>Street</th>
+                    <th>Address 1st Part</th>
+                    <th>Address 2nd Part</th>
+                    <th>City</th>
+                    <th>State</th>
+                    <th>County Province</th>
+                    <th>Postal Code</th>
+                    <th>Country</th>
+                    <th>Longitude</th>
+                    <th>Latitude</th>
+                    <th>Phone</th>
+                    <th>Website Url</th>
+                    <th>Update At</th>
+                    <th>Created At</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>@brewDetail.Id</td>
+                    <td>@brewDetail.Name</td>
+                    <td>@brewDetail.BreweryType</td>
+                    <td>@brewDetail.Street</td>
+                    <td>@brewDetail.Address2</td>
+                    <td>@brewDetail.Address3</td>
+                    <td>@brewDetail.City</td>
+                    <td>@brewDetail.State</td>
+                    <td>@brewDetail.CountyProvince</td>
+                    <td>@brewDetail.PostalCode</td>
+                    <td>@brewDetail.Country</td>
+                    <td>@brewDetail.Longitude</td>
+                    <td>@brewDetail.Latitude</td>
+                    <td>@brewDetail.Phone</td>
+                    <td><a href="@brewDetail.WebsiteUrl">@brewDetail.WebsiteUrl</a></td>

```

Comment:
I added anchor tag <a> to create a hyperlink to link to the website of the corresponding brewery. I made this change from user point of view. This will help the users to navigate directly to the website of a brewery they like for more details.
https://www.w3schools.com/tags/tag_a.asp